### PR TITLE
fix: fix some typo in README.md

### PR DESCRIPTION
**Pull request by Shouren**
* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update with typo in README.md

* **What is the current behavior?** (You can also link to an open issue here)
The introduction section of README.md contains some typo.

* **What is the new behavior (if this is a feature change)?**
Nop

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
Nop

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,21 +7,21 @@
 
 ## Introduction
 
-FEDB is a NewSQL database optimised for realtime inference and decisioning applications. These applications put real-time features extracted from multiple time windows through a pre-trained model to evaluate new data to support decision making. Existing in-memory databases cost hundreds or even thousands of milliseconds so they cannot meet the requirements of inference and decisioning applications. 
+FEDB is a NewSQL database optimised for online inferencing and decision making applications. These applications feed a pre-trained model with real-time features extracted from multiple time series windows for evaluating new data to support decision making. Existing in-memory databases cost hundreds or even thousands of milliseconds so they cannot meet the requirements of online inferencing and decisioning applications.
```
Comment (vagetablechicken) [inline README.md:10]: 'Real-Time Inference' is more suitable than 'online'?

File: README.md [hunk 2]
```diff
@@ -7,21 +7,21 @@
...
 
 - __SQL Compatible__
 
-   FEDB is compatible with most of ANSI SQL syntax. You can implement your aplications with sqlalchemy or JDBC.
+   FEDB is compatible with most of ANSI SQL syntax. You can implement your applications with SQLAlchemy or JDBC.
 
 - __Online-offline Consistency__
 
-   Machine learning applications developed by FEDB can be launched simply and ensure online and offline consistency, which greatly reduces the cost.
+   Machine learning applications powered by FEDB can be launched easily and ensure online and offline consistency, greatly reducing the cost of landing machine learning scenarios.
```
Comment (vagetablechicken) [inline README.md:24]: What is the meaning of 'landing machine learning scenarios'? The implementation? Using ML into production?

Comment (vagetablechicken) [inline README.md:24]: What is the meaning of 'landing machine learning scenarios'? The implementation? Using ML into production?

File: README.md (hunk 1 above)
Comment (Shouren) [reply README.md:10]: According to description of "面向在线推理和决策应用的NewSQL数据库" in the REAME_cn.md, i think online is more suitable than real-time

File: README.md (hunk 2 above)
Comment (Shouren) [reply README.md:24]: It meaning using the Machine learning app to support business scenarios in production

File: README.md (hunk 1 above)
Comment (vagetablechicken) [reply README.md:10]: I see, so it's the opposite of ‘offline inference’.

File: README.md (hunk 2 above)
Comment (vagetablechicken) [reply README.md:24]: This way is easier to understand, I'm wondering if you could use these words?

Comment (Shouren) [reply README.md:24]: I prefer to current modification

**Review by dl239 — APPROVED** [review]

**Review by imotai — APPROVED** [review]

**Pull request merged**